lte/gateway/configs/health.yml

```diff
@@ -21,7 +21,7 @@ state_recovery:
     - pipelined
 
   # Number of restarts of services_check that triggers recovery process
-  restart_threshold: 3
+  restart_threshold: 2
```

Comment:
what is the reason for this change?

Reply:
From the testing we've done we are tuning this to trigger clearing of corrupt state if there are more than 2 restarts over a 60sec period.